Server/mods/deathmatch/logic/lua/CLuaFunctionDefs.Server.cpp

```diff
@@ -126,6 +126,29 @@ int CLuaFunctionDefs::OOP_OutputChatBox(lua_State* luaVM)
     return 1;
 }
 
+int CLuaFunctionDefs::ClearChatBox(lua_State* luaVM)
+{
+    CElement* pElement;

```

Comment:
Uninitialized pointer ? (I see that the whole file is doing the same but is it safe to keep doing it ?)

Reply:
Argument values are initialized by CScriptArgReader

Reply:
ok my knowledge about "uninitialized pointers" was off, thanks for clearing that for me.